src/pages/Login/components/User.js

```diff
@@ -0,0 +1,203 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import GlobalStyle from '../../../styles/GlobalStyle';
+import KakaoModal from './KakaoModal';
+import * as S from './User.styled';
+
+const User = ({ content: { title, text, url, text2 } }) => {
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState({
+    id: '',
+    password: '',
+    name: '',
+    number: '',
+    checkpassword: '',
+  });
+
+  // 유효성 검사 정규식
+  const emailRegex = new RegExp(
+    /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i
+  );
+  const passwordRegex = new RegExp(
+    /^(?=.*[A-Za-z])(?=.*\d)(?=.*[~!@#$%^&*()+|=])[A-Za-z\d~!@#$%^&*()+|=]{8,16}$/
+  );
+
+  // 로그인 회원가입 폼 구분
+  const [isLogin, setIsLogin] = useState(title === '로그인');
```

Comment:
```suggestion
  const isLogin = title === '로그인'
```

해당 데이터를 꼭 state로 만들지 않아도 됩니다!